imdown-mobile/squads/squads.tsx

```diff
@@ -85,18 +90,19 @@ const Squads = (props) => {
     setAddSquadModalVisble(false)
   }
 
-  const goToEvents = (events: Event[], squadName: string, squadEmoji: string) => {
+  const goToEvents = (events: Event[], squadName: string, squadEmoji: string, email: string) => {
     props.navigation.navigate("Events", {
       events: events,
       squadName: squadName,
-      squadEmoji: squadEmoji
+      squadEmoji: squadEmoji,
+      email: email
```

Comment:
not for this PR (for fetching squads from backend), but I need squadId in order to query for events for given squad. so when you fetch squads from backend, could you also send over squadId?

Reply:
yup will do